.github/workflows/release-dev.yml

```diff
@@ -6,12 +6,16 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
+  testing:
```

Comment:
I think our Python client runs through testing on the release workflows, felt like it made sense to add it, but let me know otherwise.